nuc_morph_analysis/analyses/neighbor_of_X/example_dying_track_neighbors.py

```diff
@@ -0,0 +1,89 @@
+#%%
+from nuc_morph_analysis.lib.visualization.notebook_tools import save_and_show_plot
+from nuc_morph_analysis.lib.preprocessing import global_dataset_filtering
+import numpy as np
+from pathlib import Path
+import os
+import matplotlib.pyplot as plt
+from nuc_morph_analysis.lib.preprocessing import load_data
+from nuc_morph_analysis.lib.preprocessing import labeling_neighbors_helper
+from nuc_morph_analysis.lib.visualization.plotting_tools import plot_colorized_img_with_labels
+
+# TEMP: loading local for testing and speed
```

Comment:
Can remove load local now that you are done testing! :)

Reply:
Just kidding.. I tried loading the data freshly and running the workflows you added and I am getting a bunch of key errors. 

`KeyError: 'identified_death'` in this workflow

and in the next workflow:

`KeyError: "['has_mitotic_neighbor_breakdown', 'has_mitotic_neighbor_formation', 'has_mitotic_neighbor', 'has_mitotic_neighbor_formation_backward_dilated', 'has_mitotic_neighbor_breakdown_forward_dilated', 'has_mitotic_neighbor_dilated'] not found in axis"`

Reply:
done